runtime/tutor

```diff
@@ -1340,6 +1340,182 @@ actions available in a popup. This will allow you to:
+a list of available commands.
+
+To open a new empty buffer in a vertical split on the right half
+of your current window, you can use Ctrl-w nv (i.e., press Ctrl
+and w at the same time, then press n, then press v). Your current
+window will now split in 2 vertically, a new empty buffer split
+will appear on the right half, and your cursor will jump to the
+new vertical split.
+
+To create a new empty buffer in an horizontal split, press
+Ctrl-w ns . This will split your current window in 2 horizontally
+and create a new buffer, and your cursor will jump to the new
+horizontal split.
+
+
+
+
+=================================================================
+=                CHAPTER 13.2 MOVE BETWEEN SPLITS               =
+=================================================================
+
+Use Ctrl-w k to move to the split above your current split. Use
+Ctrl-w j to move back to the split below. Use Ctrl-w h to move to
+the split left, and Ctrl-w l to move back to the split right. To
+move to the next split (in opening order), press Ctrl-w w .
+
+You can now do whatever you want in your new buffers and splits.
+Once you are done with using your new buffer split,
+you can close it with Ctrl-w q . Move to the bottom right split
+with Ctrl-w l then Ctrl-w j, then press Ctrl-w q to close this
+specific split.
+
+You can also close all splits except the current one with Ctrl-w o .
+Open a third vertical split with Ctrl-w nv , then move to the
+leftmost split with Ctrl-w h twice, then from inside the split on
+the left press Ctrl-w o to close all except this split.
+
+
+
+=================================================================
+=               CHAPTER 13.3 SPLIT CURRENT BUFFER               =
+=================================================================
+
+You can also split the view of the current buffer: use Ctrl-w
+s (to split half way the current buffer horizontally and open
+a copy of it on the right too) and again use Ctrl-w v (to split
+half way vertically with the buffer opened on both splits).
+
+Close extra splits with Ctrl-w o to come back to a single window.
+
+
+
+
+
+
+
+
+
+
+
+
+=================================================================
+=               CHAPTER 13.4 USE COMMANDS TO SPLIT              =
+=================================================================
+
+You can also use the :vsplit (aliased to :vs for convenience) or
+the :hsplit (aliased to :hs ) to split vertically or horizontally
+a specific buffer. For example, type in the command:
+
+:vs something
+
+to open a new vertical split called something to the right. Here
+something is not an existing file, so this will open a new buffer
+under this name, but you could replace something with any file name
+to open it in a new buffer. Similarly, you can type the command:
+
+:hs some_more
+
+to open a new buffer called some_more in the lower half.
+some_more could be any file or path to open this specific file or path
+instead of a new empty buffer.
```

Comment:
```suggestion
to open a new buffer named "some_more" in the lower half.
"some_more" could be any file or path to open this specific file or path
instead of a new empty buffer.
```